packages/shfmt/build.sh

```diff
@@ -0,0 +1,21 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/mvdan/sh
+TERMUX_PKG_DESCRIPTION="A shell parser, and formatter"
+TERMUX_PKG_LICENSE="BSD 3-Clause"
+TERMUX_PKG_VERSION=3.1.1
+TERMUX_PKG_SRCURL=https://github.com/mvdan/sh/archive/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=db5d86d9caeb025e41307cb834b21d7a35e171afe5257277e74cb3e44a17ae00
+
+termux_step_make_install() {
+	cd "$TERMUX_PKG_SRCDIR"
+
+	termux_setup_golang
+
+	export GOPATH="$TERMUX_PKG_BUILDDIR"
+	mkdir -p "$GOPATH/src/github.com/mvdan"
+	ln -sf "$TERMUX_PKG_SRCDIR" "$GOPATH/src/github.com/mvdan/sh"
```

Comment:
`TERMUX_PKG_BUILD_IN_SRC=true` instead?

Reply:
I considered putting it in, but when looking at the other Go-using build.sh files for the right stuff to copy and paste, they all omit `TERMUX_PKG_BUILD_IN_SRC` and do this `cd`, `GOPATH` etc. stuff instead. After a build, ~/.termux-build/shfmt/src remains identical to the original shfmt source tarball,

Reply:
@buttaface Many Go-based programs require to be built on GOPATH to mimic structure created by `go get`.